base_multi_company/models/base.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 Tecnativa - Pedro M. Baeza
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import models
+
+
+class Base(models.AbstractModel):
+    _inherit = "base"
+
+    def _check_company(self, fnames=None):
+        """Inject as context the company of the record that is going to be compared
```

Comment:
This is not what the method does, which still is the original behavior.
Looks  more a comment than a docstring in fact :wink:

Reply:
Yes, it is exactly what it does this method override. I'm not documenting the original method, but what and why I'm overriding the method. The original behavior is documented in the original method.